src/Security/Security.php

```diff
@@ -2,17 +2,15 @@
 
 namespace SilverStripe\Security;
 
+use Page;
```

Comment:
I'll change this reference to a class reference.
`private static $page_class = Page::class;`